altlinux/sisyphus

```diff
@@ -0,0 +1,12 @@
+NAME="Sisyphus"
+VERSION="20240122"
+ID=altlinux
```

Comment:
Hi, do you want to finish this? Looks like we can name this file to `altlinux`:>

Reply:
renamed